### PR TITLE
Fixed consumption reporting for Sengled E1C-NB7

### PR DESCRIPTION
**Pull request by kimmeld**
This fixes Issue #6099

### CODE REVIEW

**Review by SwoopX — CHANGES_REQUESTED** [review]
Some minor comments to give the device the last finish 🙂

File: devices/sengled/e1c-nb7.json [hunk 1]
```diff
@@ -44,10 +44,12 @@
         },
         {
           "name": "state/alert",
+          "description": "The currently active alert effect.",
```
[the remark this answers is not in the document]
Comment (kimmeld) [reply devices/sengled/e1c-nb7.json]: Removed description as requested.

File: devices/sengled/e1c-nb7.json [hunk 2]
```diff
@@ -44,10 +44,12 @@
         },
         {
           "name": "state/alert",
+          "description": "The currently active alert effect.",
           "default": "none"
         },
         {
           "name": "state/on",
+          "description": "True when device is on; false when off.",
```
[the remark this answers is not in the document]
Comment (kimmeld) [reply devices/sengled/e1c-nb7.json]: Removed description as requested.

File: devices/sengled/e1c-nb7.json [hunk 3]
```diff
@@ -99,7 +101,19 @@
         },
         {
           "name": "state/consumption",
-          "refresh.interval": 300
+          "refresh.interval": 300,
```
[the remark this answers is not in the document]
Comment (kimmeld) [reply devices/sengled/e1c-nb7.json]: Adjusted refresh intervals to 360

**Comment by kimmeld** [comment]
I've made the required changes, hopefully this is good now 🙂

**Review by SwoopX — APPROVED** [review]
Looking good 👍

**Pull request merged**